bool is_safe
geometry_msgs/Wrench sensor_reading
geometry_msgs/Twist recovery_direction